### ISSUE TITLE
tmux support

### ISSUE THREAD

**Issue by wpalace** [joouha/euporie#118]
This program is fantastic! Thanks for creating it! 

I just started experimenting with it and noticed if I remote into a tmux session which I prefer to do, images will not display. I have kitty installed on both the host and remote machines.

This works:
ssh user@somehost -t

This does not work:
ssh user@somehost -t "tmux new-session -A -s mysesh"

![image](https://github.com/user-attachments/assets/af19e9e0-e171-4e6a-9068-60a6675f2608)

**Comment by joouha** [comment]
Hi,

Thanks for the kind words.

Only recent versions of tmux support sixel graphics (`tmux>=3.4`), and for this to work tmux has to be compiled with sixel support enabled. It doesn't support the kitty graphics protocol unfortunately.

It is possible to bypass tmux and send graphics commands directly to the terminal, but this is rather temperamental. You can try this by running euporie using the following command line options:
```bash
euporie-console --multiplexer-passthrough --force-graphics --graphics=kitty
```

(Please note that running euporie over SSH means that images may not be displayed at the correct size, as it's not always possible to detect the terminal size over SSH).

I hope that helps

**Comment by wpalace** [comment]
Thanks for the quick response. Running the command you provided gives me a blank area where the image should be; however, if I run just the command below, I get back the graphic but it is pixelated wrong (blocky): 
     euporie-console --multiplexer-passthrough
Is there any way to manually set my terminal pixel size? If so, I suspect might get it working.

**Comment by joouha** [comment]
Can you tell if it's blocky because it's using ANSI-art style graphics, or is it using kitty graphics but just very low resolution?

Do graphics work as expected for you over SSH when you are not using tmux?

**Comment by wpalace** [comment]
Yes, the graphics work as expected on Kitty when I use SSH (either with or without -t) without tmux. I cannot tell which graphics protocol is being used when I SSH to tmux and omit the --force-graphics parameters. To investigate further, I tried another terminal. It turns out that WezTerm works as expected with SSH to tmux, and I did not need to pass any parameters to euporie-console. Sweet!

My goal is to pass lines of code from neovim, where I can quickly edit with AI assist, to euporie-console in the most efficient means possible. I tried launching euporie-console from neovim's terminal mode, but the graphics did not work from there. I tried this with various terminals, both local and remote. This appears to be a neovim issue, not euporie. The best alternative I have so far is to use two tmux windows or panes and copy-paste, but this involves a lot of keystrokes. I haven't played around with WezTerm's multiplexing features yet. Do you have any suggestions on how to acheive this most-efficiently?, 

This project looks promising: https://github.com/mrjones2014/smart-splits.nvim?tab=readme-ov-file#tmux

**Comment by helf4ch** [comment]
Hi! I have the same problem, but with notebooks. I use tmux + kitty.

Here is `euporie-notebook --multiplexer-passthrough --force-graphics --graphics=kitty` (in tmux): 
![image](https://github.com/user-attachments/assets/01b58c1b-f903-485b-9375-b3f456f1f132)

Here is `euporie-notebook` with euporie config below (in tmux):
![image](https://github.com/user-attachments/assets/238e6d2c-b73d-4c17-968f-bd22a6276e01)

```
{
  "notebook": {
    "line_numbers": true,
    "tab_mode": "stack",
    "syntax_theme": "gruvbox-dark",
    "edit_mode": "vi",
    "color_scheme": "default",
    "autoinspect": true,
    "autocomplete": true,
    "autoformat": true,
    "set_cursor_shape": true,
    "expand": false,
    "wrap_cell_outputs": true,
    "enable_language_servers": true,
    "multiplexer_passthrough": true,
    "force-graphics": true,
    "graphics": "kitty",
    "show_side_bar": true,
    "show_cell_borders": true,
    "background_pattern": 4,
    "always_show_tab_bar": true
  }
}
```

Here is how it looks in just kitty with multiplexer_passthrough = false :
![image](https://github.com/user-attachments/assets/1ec43c32-d6fa-45cd-80fb-0b8d407ba160)

Notice also different colors for some reason. Maybe because my terminal and tmux has different themes.

> Can you tell if it's blocky because it's using ANSI-art style graphics, or is it using kitty graphics but just very low resolution?

It seems that it's blocky because it's using ANSI-art. I can copy it like a text and paste. Also, if I change graphics to none, I will see the same thing.

In wezterm, terminal that supports sixel, launching notebooks in tmux with multiplexer_passthrough = false shows the images, but they are for some reason flashing (appearing and disappearing) while scrolling.

I would like to see kitty + tmux result with multiplexer passthrough working right. What I missing?

I already have `set -g allow-passthrough on` in my tmux conf.

**Comment by helf4ch** [comment]
Hey! Thank you for your work! It's now looking great in my kitty + tmux setup

![Image](https://github.com/user-attachments/assets/20efe48a-dcd4-4030-a341-1e26e9bf5f1c)

**Comment by joouha** [comment]
This is now released in `v2.8.6`
